.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+
+python:
+    - "3.5"
+    - "3.6"
```

Comment:
@mrocklin  what python versions do you want me to target?

Reply:
Perhaps controversially, I suggest that we only do 3.6 for the examples